src/Shared/Const/Channel.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Aura development team - Licensed under GNU GPL
+// For more information, see license file in the main folder
+
+namespace Melia.Shared.Const
+{
+	/// <summary>
+	/// Channel constants
+	/// </summary>
+	public static class Channel
+	{
+		public const int ZoneMaxPcCount = 150;
```

Comment:
The thought of moving this magic number to a central place is certainly not wrong, but what it should probably actually be is an option in the conf files or the server db, not a const in the source.

Reply:
Should I create a *.conf file called channel.conf for that? No includes like the world.conf, or with a channel sub-folder?

Reply:
>so it restricts accessing clients when the limit is reached.

Yes, that's not a thing yet because server's don't communicate with each other yet, and the entire system of managing channels is non-existent. We don't have to go out of our way to shoehorn that in now, when it will change again soon anyway.

Where exactly that option should go isn't clear as a result either. Either a channel.conf, or a field on the servers data, which the login server has access to as well. Though we'll probably need a channel.conf either way in that case, because right now our channel"s" don't even know which channel they are.

Based on all this, I would say it would fit best into login.conf for the moment, as that's the server that needs the information, controls who goes where, whether you can actually login, etc. As long as we only have that one channel, that's the best location.